GripOpGras2.Client/wwwroot/appsettings.json

```diff
@@ -6,6 +6,20 @@
       }
     }
   },
-  "FarmMapsApiBaseUrl": "https://farmmaps.eu/",
-  "FarmMapsAccountBaseUrl": "https://accounts.farmmaps.eu/"
-}
\ No newline at end of file
+  "FarmMapsOidc": {
+    "Authority": "https://accounts.test.farmmaps.eu/",
+    "ClientId": "farmmapsdev",
+    "PostLogoutRedirectUri": "http://localhost:4200/",
+    "RedirectUri": "http://localhost:4200/cb",
+    "ResponseType": "code",
+    "DefaultScopes": [
+      "openid",
+      "profile",
+      "api"
+    ]
+  },
+  "FarmMapsBaseUrl": "https://test.farmmaps.eu/",
+  "FarmMapsApiBaseUrl": "https://test.farmmaps.eu/",
+  "FarmMapsAccountBaseUrl": "https://accounts.test.farmmaps.eu/"
+}
+
```

Comment:
Issue maken om dit afhankelijk van of het een DEV build is een niet test URL te doen